src/app/core/services/cce/prompt.service.ts

```diff
@@ -58,8 +58,7 @@ export class PromptService {
           'unitOfIssue': prompt.unit,
           'clientMutationId': '123474',
           'size': prompt.size,
-          'requesterNotes': prompt.requesterNotes || null,
-          'sharerNotes': prompt.sharerNotes || null,
+          'note': (prompt.sharing > 0) ? (prompt.sharerNotes || null) : (prompt.requesterNotes || null),
```

Comment:
just an opinion but I think the cleaner solution is updating the binding on the view template to note instead of converting here.